api4/channel_test.go

```diff
@@ -512,6 +512,34 @@ func TestViewChannel(t *testing.T) {
 	CheckNoError(t, resp)
 }
 
+func TestGetChannelUnread(t *testing.T) {
+	th := Setup().InitBasic().InitSystemAdmin()
+	defer TearDown()
+	Client := th.Client
+
+	user := th.BasicUser
+	channel := th.CreatePublicChannel()
+	app.AddUserToChannel(user, channel)
+
+	Client.Login(user.Email, user.Password)
+
+	_, resp := Client.GetChannelUnread(model.NewRandomString(16))
```

Comment:
There is already a CheckNoError test followed by checking the team id inside the channelUnread returned object.